packages/server/graphql/types/AzureDevOpsWorkItem.ts

```diff
@@ -23,6 +23,14 @@ const AzureDevOpsWorkItem = new GraphQLObjectType<any, GQLContext>({
         return AzureDevOpsIssueId.join(instanceId, teamProject, id)
       }
     },
+    issueKey: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'The id of the issue from Azure, e.g. 7',
+      resolve: async ({id}: {id: string}) => {
+        const {issueKey} = AzureDevOpsIssueId.split(id)
```

Comment:
wait a minute... L21 shows that `source.id === `issueKey` and this should that `source.id` is the composite key. gotta test this one out...

Reply:
ah, the split method was written to return the issueKey if the full ID didn't contain any colons. Neat!
Gonna change this to `return id` for simplicity sake